cpp/ps0005/p0005.cpp

```diff
@@ -12,6 +13,8 @@ int main() {
     cin >> num1;
     cout << "Enter the second number: ";
     cin >> num2;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input